pkg/agent/handler/traceaccessrequest_test.go

```diff
@@ -94,9 +95,9 @@ func TestTraceAccessRequestTraceHandler(t *testing.T) {
 	c.getRI = &v1.ResourceInstance{
 		ResourceMeta: v1.ResourceMeta{
 			Metadata: v1.Metadata{
-				ID: "subscription",
+				ID: "subscription-id",
```

Comment:
Should these be constants?

Reply:
Its test data, but have other string that I can create constants for. Will do the change